### PR TITLE
Chinese translation

### PR DESCRIPTION
**Pull request by felixqu**
Added Simplified Chinese translation.

### CODE REVIEW

**Comment by kspearrin** [comment]
Thanks for this. Could you also add the store translations as well? See https://github.com/bitwarden/browser/pull/32/files for example.

**Comment by kspearrin** [comment]
Also, the `description`s should remain in English. They are only for reference.

**Comment by felixqu** [comment]
Sure, I will add the store translations and recommit the messages.json file.

File: store/zh-CN/CAPTIONS.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Tile
+
+A secure and free password manager for all of your devices
```
Comment (kspearrin) [inline store/zh-CN/CAPTIONS.md]: Translation here?

File: store/zh-CN/COPY.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Name
+
+bitwarden ?Free Password Manager
```
Comment (kspearrin) [inline store/zh-CN/COPY.md]: Translation here?

**Comment by felixqu** [comment]
Is there anything wrong？

**Comment by kspearrin** [comment]
Hey sorry, I never saw your update to my initial review. Looks good.

**Pull request merged**